src/modules/entities/modules/reports/services/actions/addNote.js

```diff
@@ -0,0 +1,3 @@
+import { apiRequestActions } from '@ackee/redux-utils';
+import { AddNoteTypes } from './types';
+export default apiRequestActions(AddNoteTypes, { isDetailRequest: true });
```

Comment:
this is also not good usage of this apiRequestActions - this should be only for ids, but we can leva it for now - it is not that bad as sending those filters/params in previous MR